website/addons/dataverse/static/dataverseUserConfig.js

```diff
@@ -123,6 +159,26 @@ function ViewModel(url) {
             }, timeout);
         }
     };
+
+    // Update observables with data from the server
+    $.ajax({
```

Comment:
Requests should not be made in a constructor. Instead, explicitly call a method to update the view model from server data.

```js
// in the page module

var dataverseViewModel = new DataverseViewModel(url);
$osf.applyBindings(...)
dataverseViewModel.fetch();
```